site/sfguides/src/getting_started_with_devops_in_snowflake/getting_started_with_devops_in_snowflake.md

```diff
@@ -0,0 +1,582 @@
+author: Vincent Raudszus
+id: getting_started_with_devops_in_snowflake
+summary: Getting Started with DevOps in Snowflake
+categories: devops,getting-started,data-engineering
+environments: web
+status: Published
+feedback link: <https://github.com/Snowflake-Labs/sfguides/issues>
+tags: Getting Started, Data Science, Data Engineering, Twitter 
+
+# Getting Started with DevOps in Snowflake
+
+<!-- ------------------------ -->
+## Overview
+
+Duration: 5
+
+A data pipeline is a series of automated processes that ingest, process, and move data from one place to another. It's a crucial component in data management and analytics, enabling organizations to make use of the vast amounts of data available to them.
+
+The data engineer’s work doesn’t end when the pipeline is up and running for the first time. As businesses grow and change, so do their data needs. Data schemas as well as the pipeline as a whole need to evolve to capture new data sources or analyst requirements. In software engineering, DevOps techniques are applied to emphasize collaboration and communication to make changes rapidly and reliably. Similarly, DevOps best practices should be followed to ensure that changes to tables, data pipelines, and other database objects are made in a controlled, repeatable, and safe manner.
+
+Are you interested in bringing your database game to the next level by unleashing the power of Snowflake’s many DevOps capabilities, empowering you to build safe and robust data pipelines? Well then, this Quickstart is for you!
+
+This quickstart will cover a lot of ground. By the end of this guide, you will have built a robust CI/CD pipeline that automates the deployment of changes to a data pipeline from Git, promotes the separation of dev and prod databases and facilitates monitoring. You'll also learn how to use Snowflake's marketplace to enrich your data without incurring storage costs, get to know the new developer CLI tool and the Visual Studio Code extension!
+
+![DevOps Overview](assets/devops-overview.png)
+
+The data pipeline used as an example in this quickstart makes use of data available on the Snowflake marketplace to come up with the perfect spot for a family vacation. Let's get started!
+
+> aside negative
+> Note: The Git integration, Create or Alter and Jinja templating are still in preview.
```

Comment:
wording sounds off. Do we have standard wording for preview disclaimers?

Reply:
@iamontheinet